tests/lua-tests/src/controller.lua

```diff
@@ -10,8 +10,10 @@ require "cocos.init"
 
 local director = cc.Director:getInstance()
 local glView   = director:getOpenGLView()
+local widthx = 1024
+local heighty = 2112
```

Comment:
Why do we need to change this width and height for other platforms (like windows, android)?

Reply:
reverted